### DISCUSSION TITLE
failover模式测延时问题

### DISCUSSION THREAD

**Discussion by peopleel** [eycorsican/leaf#75] [Q&A] [answered]
在failover 模式下 健康检查开启，测试是tcp，还是icmp，测试地址能否自定义？tcp测延时与实际使用出入较大，望大佬解答，谢谢！

**Comment by eycorsican** [comment, marked as answer]
TCP 是 HTTP 请求的 RTT，UDP 是一个 DNS 请求的 RTT

**Reply by peopleel** [reply]
恩恩，多谢解答。能否加入服务器真连接延迟测试

**Reply by eycorsican** [reply]
你仔细描述下要怎么测？

**Reply by peopleel** [reply]
用代理访问www.google.com/generate_204，所用时间。这个样使用fail模式优选代理的时候，更接近优选速度连接好的。有的时候用tcping值低，但是速度很慢。不知道我有没有说明白。

**Reply by eycorsican** [reply]
这就是我指的 HTTP 请求 RTT，目前就是使用这种检测方式

**Reply by peopleel** [reply]
恩恩，感谢大佬解答！
